### ISSUE TITLE
[Bug] QPullToRefresh not working on mobile (mobile web, cordova, capacitor)

### ISSUE THREAD

**Issue by josephharveyangeles** [quasarframework/quasar#6130]
**Describe the bug**
QPullToRefresh is not working on mobile. Getting the following error:
```
Uncaught TypeError: Cannot read property 'event' of undefined
    at shouldStart (touch.js?3627:94)
    at HTMLDivElement.touchStart (TouchPan.js?75c3:158)
```

**Codepen/jsFiddle/Codesandbox (required)**
Open the any of the pens on quasar docs: [QPullToRefresh](https://quasar.dev/vue-components/pull-to-refresh), open it on mobile.

**To Reproduce**
1. Go to [https://quasar.dev/vue-components/pull-to-refresh](https://quasar.dev/vue-components/pull-to-refresh)
2. Open Chrome Dev Tools
3. Try to Pull to refresh
4. See error.

**Expected behavior**
Should trigger the action and no error should occur.

**Platform (please complete the following information):**
OS:
Node: v12.4.0
NPM: 6.9.0
Yarn: 1.21.1
Browsers: Chrome
iOS:
Android:
Electron:

**Additional context**
There are no issues when running it as SPA or just plain web. Problem arise when it's on mobile, cordova, capacitor albeit just plain mobile web.

**Comment by rstoenescu** [comment]
Fix already available in "quasar" v1.7.4 (to be released in half an hour).

**Comment by josephharveyangeles** [comment]
Oh wow that's really fast. Thanks Razvan!
